frontend/device/generic/device.lua

```diff
@@ -116,8 +116,13 @@ end
 -- ONLY used for Kobo and PocketBook devices
 function Device:onPowerEvent(ev)
     local Screensaver = require("ui/screensaver")
+    local network_manager = require("ui/network/manager")
     if (ev == "Power" or ev == "Suspend") and not self.screen_saver_mode then
         self.powerd:beforeSuspend()
+        if network_manager.wifi_was_on then
```

Comment:
@S-trace I mean this one here, not the one in resume.

Onlookers see https://github.com/koreader/koreader/issues/3626#issuecomment-359479082